backend/models/contributions.js

```diff
@@ -0,0 +1,50 @@
+const bcrypt = require('bcrypt-nodejs');
+const User = require(`./users.js`);
+
+module.exports = (sequelize, DataTypes) => {
+  const Contribution = sequelize.define('user', {
+    //bute_id that will be used for individual contributions
+    //TODO: tell postgres to generate unique uuid
+    bute_id: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      unique: true,
+      validate: {
+        notEmpty: true,
+        isUUID: true, //sequelize validation for uuid
+      },
+    },
+    data: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: false,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    // author: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    //   unique: true,
+    //   validate: {
+    //     notEmpty: true,
+    //     isEmail: true, // sequelize validation
+    //   },
+    // },
+    //the dynamic array linking to contribution table
+    prev_versions: {
+      type: Contribution,
+      allowNull: true, //may be init_bute
+      validate: {
+        isArray: true,
+      },
+    },
+    date_and_time: {
+      type: DataTypes.DATE,
+    },
+    reviewers: {
```

Comment:
This also needs to be changed. Since we can no longer refer back to the entries in the `User` table.

Something along the lines of `Contribution.hasMany(User, {as: 'Reviewers'});`

Double check tho

Reply:
For now, delete the `reviews` column. It'll be handled automatically when we fix #17